### DISCUSSION TITLE
Automazione - Si può fare?

### DISCUSSION THREAD

**Discussion by Pink1881** [indomus/forum#2833] [GENERALE - Discussioni generiche]
Ciao,
volendo domotizzare il portone del garage ho usato un Sonoff 4CHPROR3 per gestire il telecomando rolling code e l'ho programmato perchè al comando di attivazione si "prema" il pulsante relativo e poi il contatto si stacchi automaticamente e funziona tutto perfettamente.
Ora vorrei che quando do il comando la prima volta (ed il portone si apre) si accenda anche la luce del garage mentre al secondo comando questa si spenga.
Come potrei fare?
Servirebbe un interruttore attivabile dallo stesso contatto pulito del Sonoff ma non so su cosa orientarmi (o se c'è qualcosa di adatto).

**Comment by effe-7** [comment]
Ciao, non se usi Hass o qualche hub.
Io penserei di fare una automazione con la condizione che quando premi il comando se la luce è spenta si accende e se la luce è accesa si spegne per semplificare fai due automazioni

**Reply by Pink1881** [reply]
Interessante, non sapevo si potesse fare qualcosa del genere, grazie.
Ma cosa comporta?
Cosa dovrei comprare e fare?
